src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -518,4 +520,32 @@ def remove_file_post
       end
     end
   end
+
+  describe 'GET #trigger_services' do
+    let(:post_url) { "http://localhost:3200/source/#{source_project}/#{service_package}?cmd=runservice&user=#{user}" }
```

Comment:
It is like that everywhere:

- https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/features/webui/packages_spec.rb#L104

- https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/models/package_spec.rb#L39

- https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/models/service_spec.rb#L9

What is the alternative?